Sources/en.lproj/Localizable.strings

```diff
@@ -197,6 +197,7 @@
 // Settings storyboard
 "settings.tab-bar.item" = "Settings";
 "settings.sections.information" = "Information";
+"settings.sections.discover" = "Discover";
```

Comment:
Adding this has changed the header for the English language, and this has to be changed for all the languages.